spec/support/aruba.rb

```diff
@@ -1,9 +1,32 @@
 require 'aruba/rspec'
 require 'arel'
+require 'rake'
+require 'rails'
 
 module ChronoTest
 
   module Aruba
+
+    def load_schema_task(as_regexp = false)
```

Comment:
let's use keyword arguments, `load_schema_task(as_regexp: false)` to improve this method call: `load_schema_task(true)` is not clear to understand